packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -654,6 +673,23 @@ const setOutputValues = async () => {
 	optimizationResult.value = optimzationResult;
 };
 
+const preProcessedInterventionsData = computed<
```

Comment:
I want to group and format the data for the interventions chart here...interventions such as beta can be created in separate cards, so I want to group them all together to be used in 1 chart